src/components/Network.vue

```diff
@@ -59,7 +59,6 @@ class Network extends Vue {
                     new Nimiq.PublicKey(keyguardResult.publicKey),
                     new Nimiq.Signature(keyguardResult.signature),
                 ).serialize(),
-                keyguardRequest.networkId,
```

Comment:
I don't think this is related to page refresh, is it?

Reply:
It is, because I am updating the KeyguardClient dependency, to have the RPC features required, but the API there changed and does not include the networkId anymore. Thus, to use the new KeyguardClient, this needs to be removed (will otherwise throw a typescript error).